graylog2-web-interface/src/views/logic/views/OnSaveViewAction.js

```diff
@@ -1,8 +1,11 @@
 import UserNotification from 'util/UserNotification';
 import { ViewManagementActions } from 'views/stores/ViewManagementStore';
+import { ViewActions } from 'views/stores/ViewStore';
 
-export default (view) => {
-  return ViewManagementActions.update(view)
-    .then(() => UserNotification.success(`Saving view "${view.title}" was successful!`, 'Success!'))
-    .catch(error => UserNotification.error(`Saving view failed: ${error}`, 'Error!'));
+export default (newView) => {
+  return ViewActions.update(newView).then(({ view }) => {
+    return ViewManagementActions.update(view);
+  }).then(({ title }) => {
+    return UserNotification.success(`Saving view "${title}" was successful!`, 'Success!');
```

Comment:
Do you prefer using the view title, returned from the database, or the initial view title we passed to the `onSaveViewAction`?